### PR TITLE
Add reference to education-notebook

### PR DESCRIPTION
**Pull request by damianavila**
New community image living at https://github.com/umsi-mads/education-notebook.

Now we have 2 examples!!

### CODE REVIEW

File: docs/using/selecting.md [hunk 1]
```diff
@@ -142,8 +142,10 @@ You must refer to git-SHA image tags when stability and reproducibility are impo
 
 The core stacks are just a tiny sample of what's possible when combining Jupyter with other technologies. We encourage members of the Jupyter community to create their own stacks based on the core images and link them below.
 
-*This list only have 1 example. You can be the next!*
+*This list only have 2 examples. You can be the next!*
```
Comment (parente) [inline docs/using/selecting.md]: Small Grammar change: `has 2 examples`

Comment (parente) [inline docs/using/selecting.md]:
```suggestion
*This list only has 2 examples. You can be the next!*
```

**Comment by parente** [comment]
XRef #799

File: docs/using/selecting.md (hunk 1 above)
Comment (damianavila) [reply docs/using/selecting.md]: Nice, I never used the suggestion more... interesting!

**Pull request merged**